weave/__init__.py

```diff
@@ -22,8 +22,8 @@
 from weave.trace.util import Thread as Thread
 from weave.trace.util import ThreadPoolExecutor as ThreadPoolExecutor
 from weave.type_handlers.Audio.audio import Audio
-from weave.type_handlers.File.file import File
```

Comment:
There are probably few/no people using this, but in case there are it might be nice to have a deprecation message